src/main/java/core/basesyntax/model/figure/Triangle.java

```diff
@@ -1,6 +1,15 @@
 package core.basesyntax.model.figure;
 
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity(name = "Triangle")
 public class Triangle extends Figure {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
```

Comment:
the same?

Reply:
but it's normal that I see compilation error, like this
![image](https://user-images.githubusercontent.com/59320919/107206326-8c887300-6a07-11eb-87d4-31f80fc3b289.png)
![image](https://user-images.githubusercontent.com/59320919/107206341-93af8100-6a07-11eb-9e2d-9914dd4ca93a.png)

Reply:
It's ok, because we have id at the Figure class

Reply:
you can try to add getter and setter for id field